workspaces/liferay-sample-workspace/client-extensions/liferay-sample-fds-cell-renderer/tsconfig.json

```diff
@@ -11,7 +11,9 @@
 		"outDir": "./build",
 		"resolveJsonModule": true,
 		"sourceMap": false,
-		"target": "ES2020"
+		"target": "ES2020",
+		"types": [
```

Comment:
Is `types` supposed to be empty?

Reply:
I would say yes, since there are no types involved in the project and, by default, `tsc` usually tries to get them from several places which can lead to problems.

I guess that's what happened to @allen-ziegenfus , right?